src/components/SearchResults/SearchResults.scss

```diff
@@ -0,0 +1,84 @@
+$highlight-background: #f2f8fb;
+
+.search-result-card {
+    margin: 20px 0px 20px 0px;
+    padding: 10px 30px 0px 30px;
+    text-align: left;
+    border-radius: 4px;
+    background-color: #ffffff;
+    border: solid 1px #d6dbe0;
+
+    &:hover{
+        background-color: $highlight-background;
+        cursor: pointer;
+    }
+}
+
+.card-link {
+    a:hover {
+        text-decoration: none;
+    }
+}
+
+.result-icon {
+    font-size: 21px;
+    line-height: 1.67;
+    color: #414141;
+}
+
+.result-title {
+    padding-left: 8px;
+    font-family: OpenSans;
+    font-size: 20px;
+    font-weight: 500;
+    color: #005686;
```

Comment:
We've already defined this color as both the `$primary` and `$deep-sea-blue`. If this is a color that should change with theming, use `$primary`, if it should always be blue (probably in this case) use `$deep-sea-blue`.